src/main/java/net/minecraftforge/fluids/FluidUtil.java

```diff
@@ -122,11 +121,12 @@ public static ItemStack tryFillContainer(ItemStack container, IFluidHandler flui
      * @param player           Player for making the bucket drained sound. Pass null for no noise.
      * @param doDrain          true if the container should actually be drained, false if it should be simulated.
      * @return The empty container if successful, null if the fluid handler couldn't be filled.
-     *         NOTE The empty container will have a stackSize of 0 when a filled container is consumable,
-     *              i.e. it has a "null" empty container but has successfully been emptied.
+     * NOTE The empty container will have a stackSize of 0 when a filled container is consumable,
+     * i.e. it has a "null" empty container but has successfully been emptied.
      */
     @Nullable
-    public static ItemStack tryEmptyContainer(ItemStack container, IFluidHandler fluidDestination, int maxAmount, @Nullable EntityPlayer player, boolean doDrain)
+    public static ItemStack tryEmptyContainer(ItemStack container, IFluidHandler fluidDestination, int maxAmount, @Nullable EntityPlayer player,
+            boolean doDrain)
```

Comment:
Thanks for catching that! I must've missed this line when I double-checked the formatting.